server/src/js/components/map/live.js

```diff
@@ -135,7 +135,7 @@ function LiveMap (mapId, options) {
       // Stations
       if (props.status === 'Suspended' || props.status === 'Closed' || (!props.value && !props.iswales)) { // Any station that is closed or suspended
         state = 'error'
-      } else if (props.value && props.atrisk && props.type !== 'C') { // Any station (excluding sea levels) that is at risk
+      } else if (props.value && props.atrisk && props.type !== 'C' && !props.iswales) { // Any station (excluding sea levels) that is at risk
```

Comment:
The comment here is now (slightly) misleading. I guess this is my issue with comments in code.